src/Entities/Entity.cpp

```diff
@@ -518,7 +518,7 @@ bool cEntity::DoTakeDamage(TakeDamageInfo & a_TDI)
 
 			if (Random.RandBool(Chance / 100.0))
 			{
-				a_TDI.Attacker->TakeDamage(dtAttack, this, 0, Random.RandInt(1, 4), 0);
+				a_TDI.Attacker->TakeDamage(dtAttack, this, 0, Random.RandReal(1.0f, 4.0f), 0);
```

Comment:
Isn't raw damage usually done in increments of 1 hp? I think just casting the result of `RandInt` makes more sense.

Reply:
I can't find any reference to this, and since the UI always rounds to the nearest half-heart, I don't think this matters much.